command/tokenize.go

```diff
@@ -110,3 +128,61 @@ type TokenizeError struct {
 func (e TokenizeError) Error() string {
 	return fmt.Sprintf(e.Text, e.Position)
 }
+
+// quotationMarkCategory returns a pointer to the range of unicode characters equivalent to the given quote, or nil if
+// the given character isn't a quotation mark.
+func quotationMarkCategory(r rune) *unicode.RangeTable {
+	switch {
+	case unicode.In(r, singleQuotes):
+		return singleQuotes
+	case unicode.In(r, doubleQuotes):
+		return doubleQuotes
+	default:
+		return nil
+	}
+}
+
+// isMatchingQuotationMark returns whether a quotation mark is in the given *RangeTable, either singleQuote or
+// doubleQuote.
+func isMatchingQuotationMark(r rune, other *unicode.RangeTable) bool {
+	return unicode.In(r, other)
+}
+
+// getBasicQuote accepts either the singleQuote or doubleQuote *RangeTable, and returns the ascii representative of that
+// category (either ' or ").
+func getBasicQuote(table *unicode.RangeTable) rune {
+	switch table {
+	case singleQuotes:
+		return '\''
+	case doubleQuotes:
+		return '"'
+	default:
+		return rune(0)
+	}
+}
+
+// unescape translates a given printable character rune into it's corresponding control character.
+func unescape(r rune) (rune, error) {
+	switch r {
```

Comment:
1. Can we sort these somehow?
2. Per [this reference](https://www.ibm.com/docs/en/zos/2.3.0?topic=set-escape-sequences), this list should also include:
   1. \\' (single quote)
   2. \\? (question mark)